app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
 
     defaultConfig {
         applicationId "pl.rspective.voucherify.android.demo"
-        minSdkVersion 10
```

Comment:
There was a piece of code that didn't compile with API Level 10. It was the constructor of `LinearLayout` that didn't accept a 3rd parameter `int defStyle`. I removed it and reverted `minSdkVersion` to 10.